Cargo.toml

```diff
@@ -19,8 +22,20 @@ sha3 = "0.10.0"
 bincode = "1.3.3"
 rand = "0.8.5"
 thiserror = "1.0"
+num-rational = "0.2"
+num-traits = "0.2"
 num-bigint = "0.4"
 num-integer = "0.1"
+schnorrkel = "0.10.2"
+async-trait = "0.1.52"
+secrecy = "0.8.0"
+futures-timer = "3.0.2"
+log = "0.4.14"
+merlin = "3.0.0"
+rocksdb = "0.18.0"
+anyhow = "1.0"
```

Comment:
Since bazuka will be a library with solid code, I feel like using something like `thiserror` would make more sense.